summoner.cabal

```diff
@@ -91,6 +91,7 @@ test-suite summoner-test
                      , relude
                      , tasty
                      , tasty-hedgehog
+                     , tomland
                      , summoner
   default-extensions:  NoImplicitPrelude
```

Comment:
@willbasky One more thing: in your previous PR you've introduced default warnings to generated `.cabal` files. Could you also add those warnings to `summoner.cabal` file and fix any appear warnings? I think we should test warnings that we generated. It probably should be done under previous PR but let's do this here.